storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -1,51 +1,51 @@
 :_content-type: ASSEMBLY
 [id="persistent-storage-csi-ebs"]
-= AWS Elastic Block Store CSI Driver Operator
+= Amazon Elastic Block Store CSI Driver Operator
 include::_attributes/common-attributes.adoc[]
 :context: persistent-storage-csi-ebs
 
 toc::[]
 
 == Overview
 
-{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for AWS Elastic Block Store (EBS).
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Amazon Elastic Block Store (EBS).
 
 Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a Container Storage Interface (CSI) Operator and driver.
 
-To create CSI-provisioned PVs that mount to AWS EBS storage assets, {product-title} installs the AWS EBS CSI Driver Operator and the AWS EBS CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
+To create CSI-provisioned PVs that mount to Amazon EBS storage assets, {product-title} installs the Amazon EBS CSI Driver Operator and the Amazon EBS CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
```

Comment:
Revoked.

Reply:
@smunje1 It doesn't seem like this title is linking out to OCP resources or code that reference "AWS EFS". "Amazon EFS CSI driver" is the appropriate name to be used for the driver. Is there a reason I'm missing why this update should be revoked?

https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html

Reply:
Updated, "To create CSI-provisioned PVs that mount to AWS EBS storage assets, {product-title} installs the [AWS EBS CSI Driver Operator](https://github.com/openshift/aws-ebs-csi-driver-operator) (a Red Hat operator) and the AWS EBS CSI driver by default in the openshift-cluster-csi-drivers namespace."